Cargo.toml

```diff
@@ -59,8 +71,21 @@ futures-lite = "2.3.0"
 hex = "0.4.0"
 hickory-resolver = "0.24"
 humansize = "2"
-image = { version = "0.25.1", default-features=false, features = ["gif", "jpeg", "ico", "png", "pnm", "webp", "bmp"] }
-iroh = { version = "0.4.2", default-features = false }
+image = { version = "0.25.1", default-features = false, features = [
+  "gif",
+  "jpeg",
+  "ico",
+  "png",
+  "pnm",
+  "webp",
+  "bmp",
+] }
+iroh_old = { package = "iroh", version = "0.4.2", default-features = false }
+iroh = { version = "0.15.0", default-features = false }
```

Comment:
you shouldn't need this, given you are using the individual iroh crates directly